src/make_ntuples.c

```diff
@@ -128,20 +141,19 @@ int run(char * in_filename, bool use_fmt, bool debug, int nevn, int run_no, doub
             double chi2 = rtrk.chi2   ->at(pos);
             double ndf  = rtrk.ndf    ->at(pos);
 
-            // Fill TNtuples. TODO. This probably should be implemented more elegantly.
-            // NOTE. If adding new variables, check their order in S_VAR_LIST.
+            // Filling the TNtuple
             Float_t v[VAR_LIST_SIZE] = {
                 (Float_t) run_no, (Float_t) evn, (Float_t) beam_E,
                 (Float_t) p.pid, (Float_t) status, (Float_t) p.q, (Float_t) p.mass, (Float_t) p.vx,
-                        (Float_t) p.vy, (Float_t) p.vz, (Float_t) p.px, (Float_t) p.py,
-                        (Float_t) p.pz, (Float_t) P(p), (Float_t) theta_lab(p),
-                        (Float_t) phi_lab(p), (Float_t) p.beta,
+                (Float_t) p.vy, (Float_t) p.vz, (Float_t) p.px, (Float_t) p.py,
+                (Float_t) p.pz, (Float_t) P(p), (Float_t) theta_lab(p),
+                (Float_t) phi_lab(p), (Float_t) p.beta,
                 (Float_t) chi2, (Float_t) ndf,
                 (Float_t) pcal_E, (Float_t) ecin_E, (Float_t) ecou_E, (Float_t) tot_E,
                 (Float_t) htcc_nphe, (Float_t) ltcc_nphe,
                 (Float_t) tof,
-                (Float_t) Q2(p, beam_E), (Float_t) nu(p, beam_E), (Float_t) Xb(p, beam_E),
-                        (Float_t) W2(p, beam_E)
+                (Float_t) Q2(p, beam_E), (Float_t) nu(p, beam_E), (Float_t) Xb(p, beam_E), (Float_t) W2(p, beam_E),
+                (Float_t) zh(p, p_el, beam_E),(Float_t) Pt2(p, p_el, beam_E),(Float_t) Pl2(p, p_el, beam_E),(Float_t) phi_pq(p, p_el, beam_E),(Float_t) theta_pq(p, p_el, beam_E)
```

Comment:
Max line length is 100 characters.

Reply:
Maybe I should write a styleguide convention and forbid code that doesn't follow it to be merged? That'd be tortuously fun...